tb/cocotb/soc/test_axil_to_apb.py

```
@@ -0,0 +1,292 @@
+# ---------------------------------------------------------------------------
+# T4 — AW arrives before W: bridge buffers AW and waits for W
+# ---------------------------------------------------------------------------
+@cocotb.test()
+async def test_aw_before_w(dut):
+    """
+    Drive AW channel one cycle before W channel.  The bridge must:
+      - Accept AW (awready) when AW is presented (state=IDLE, aw_cap clear)
+      - Hold off APB SETUP until W also arrives
+      - Complete write with bresp OKAY
+    """
+    m = await _setup(dut)
```

Flake8 (7.3.0)

[error] local variable 'm' is assigned to but never used

(F841)